plugins/chat/assets/javascripts/discourse/components/chat-thread.hbs

```diff
@@ -4,36 +4,14 @@
   {{did-insert this.loadMessages}}
 >
   <div class="chat-thread__header">
-    <div class="chat-thread__info">
-      <div class="chat-thread__title">
-        <h2>{{this.title}}</h2>
-
-        <LinkTo
-          class="chat-thread__close"
-          @route="chat.channel"
-          @models={{this.chat.activeChannel.routeModels}}
-        >
-          {{d-icon "times"}}
-        </LinkTo>
-      </div>
-
-      <p class="chat-thread__om">
```

Comment:
Why remove all this and the title when this stuff was needed in the design? Are we just going to add it back later?